File: test/samples/sample.js
function classify(a, b) {
  if (a > 0 && b > 0) {
    return 1;
  } else if (a < 0) {
    return 2;
  } else {
    return 3;
  }
}

function loop(n, m) {
  let total = 0;
  for (let i = 0; i < n; i++) {
    while (total < m) {
      if (total === 7) {
        total = total + 2;
      }
      total = total + 1;
    }
  }
  return total;
}

File: test/samples/sample.ml
let classify a b =
  if a > 0 && b > 0 then 1
  else if a < 0 then 2
  else 3

let loop n m =
  let total = ref 0 in
  for _i = 1 to n do
    while !total < m do
      if !total = 7 then total := !total + 2;
      total := !total + 1
    done
  done;
  !total
